ibmnode/8/sdk/linux/x86_64/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM ubuntu:16.04
+
+MAINTAINER Bethany Griggs <bethany.griggs@uk.ibm.com> (@BethGriggs)
+
+RUN buildDeps='wget ca-certificates unzip openjdk-8-jre'; \
+    apt-get update \
+    && apt-get install -y --no-install-recommends \
+    $buildDeps \
+    && rm -rf /var/lib/apt/lists/* \
+    set -eux; \
+    IBM_VERSION="8.1.4.0" \
+    NODE_VERSION="8.1.4" \
+    ARCH="$(dpkg --print-architecture)"; \
+    case "${ARCH}" in \
+       amd64|x86_64) \
+         SHA256="$IBM_VERSION/linux/x86_64/ibm-$IBM_VERSION-node-v$NODE_VERSION-linux-x64.bin.sha256.txt"; \
+         BINARY="$IBM_VERSION/linux/x86_64/ibm-$IBM_VERSION-node-v$NODE_VERSION-linux-x64.bin"; \
```

Comment:
You'd need to have the sha256 sum embedded directly into the Dockerfile itself as both the binary and the sha256sum cannot be from the same source. (Infact this is a requirement if you want to submit this to be an official docker image)